requirements.txt

```diff
@@ -7,3 +7,4 @@ argparse==1.1
 fastchardet==0.2.0
 mock==1.0.0
 simplejson==3.8.0
+spidermonkey==41.0a2.post1
```

Comment:
Done: https://pyrepo.addons.mozilla.org/spidermonkey-41.0a2.post1.tar.gz

Not sure we need the wheel for now. It would be a nice speedup, but if it's a pain to create... travis could do it for us, but if it can't be uploaded to pypi...